### DISCUSSION TITLE
Fryer with my spotify account?

### DISCUSSION THREAD

**Discussion by thedinz** [RandomNinjaAtk/arr-scripts#225] [Q&A] [answered]
Its looks like fryer supports spotify, where as the extended config only asks for tidal or deezer tokens.

Is there a way to use spotify?

**Comment by RandomNinjaAtk** [comment, marked as answer]
I thought fryer uses YouTube on the backend. So not sure how that would be useful?

**Reply by thedinz** [reply]
Hello, thank you for the reply, this came about because i clicked the fryer link under credits on the lidarr extended page

https://github.com/miraclx/freyr-js

That page says "Download songs from Spotify, Apple Music and Deezer." and then it says you can enter your sporify credentials in the config file but i searched and i dont see a fryer config file in my extended folder.

**Reply by RandomNinjaAtk** [reply]
But if the files are sourced from YouTube, there isn’t anything beneficial by logging into Spotify, so I’m not sure how that would be helpful or useful…

**Reply by thedinz** [reply]
Now i see what your saying. Just tryign to save money on a subscription, i prefer spotify but im using deezer just for lidarr, seems like a waste because i dont really use the server except for lidarr lol. Oh well, i got some stuff to figure out then it seems.

**Reply by RandomNinjaAtk** [reply]
If you’re happy with Freyr, just set it deezer and don’t input a ARL token to use freyr.
